Software/app/src/main/java/org/foi/emp/collegecoursemanager/DodavanjeKolegija.java

```diff
@@ -0,0 +1,114 @@
+                        final int idElementa1 = unosElementaModelaPracenja(maksimalnibrojBodovaKol2, nazivEl1);
+                        unosElementaNaModeluPracenja(idElementa1, idModela);
+
+                        final String nazivEl2 = "Laboratorijske vjezbe";
+                        final int idElementa2 = unosElementaModelaPracenja(maksimalnibrojBodovaLab, nazivEl2);
+                        unosElementaNaModeluPracenja(idElementa2, idModela);
+
+                        final String nazivEl3 = "Aktivnost";
+                        final int idElementa3 = unosElementaModelaPracenja(maksimalnibrojBodovaAktivnost, nazivEl3);
+                        unosElementaNaModeluPracenja(idElementa3, idModela);
+                        Toast.makeText(getApplicationContext(), "Uspješno ste unijeli kolegij", Toast.LENGTH_SHORT).show();
+                    } else {
+                        Toast.makeText(getApplicationContext(), "Zbroj mogucih bodova nije jednak 100", Toast.LENGTH_SHORT).show();
+                    }
+                } else
+                    Toast.makeText(getApplicationContext(), "Mjesta za unos moraju biti popunjena", Toast.LENGTH_SHORT).show();
+            }
+        });
+    }
+
+    private int unosModelaPracenja(String nazivModela) {
+        ModelPracenja noviModel = new ModelPracenja();
+        noviModel.setNaziv(nazivModela);
+        return (int) Database.getInstance(getApplicationContext()).getModelPracenjaDAO().unosModelaPracenja(noviModel)[0];
+    }
+
+    private void unosKolegija(int idModela, String naziv) {
+        Kolegij noviKolegij = new Kolegij();
+        noviKolegij.setModelPracenja(idModela);
+        noviKolegij.setNazivKolegija(naziv);
+        Database.getInstance(getApplicationContext()).getKolegijDAO().unosKolegija(noviKolegij);
+    }
+
+    private int unosElementaModelaPracenja(int maksimalnibrojBodovaKol1, String nazivElementa) {
+        ElementModelaPracenja emp = new ElementModelaPracenja();
+        emp.setMaksimalniBrojBodova(maksimalnibrojBodovaKol1);
+        emp.setNaziv(nazivElementa);
+        emp.setOstvareniBodovi(0);
+        return (int) Database.getInstance(getApplicationContext()).getModelPracenjaDAO().unosElementaModelaPracenja(emp)[0];
+    }
+
+    private void unosElementaNaModeluPracenja(final int idElementa, final int idModela) {
+        ElementiNaModeluPracenja empi = new ElementiNaModeluPracenja();
+        empi.setElementModelaPracenja(idElementa);
+        empi.setModelPracenja(idModela);
+        Database.getInstance(getApplicationContext()).getModelPracenjaDAO().unosElementaNaModeluPracenja(empi);
+    }
+
+    private boolean provjeraPraznihUnosa(String kol1, String kol2, String aktivnost, String labosi, String nazivKolegija, String nazivModela) {
+        if (kol1.isEmpty() || kol2.isEmpty() || aktivnost.isEmpty() || labosi.isEmpty() || nazivKolegija.isEmpty() || nazivModela.isEmpty()) {
+            return false;
+        }
+        return true;
```

Comment:
If you're having StringUtils class in project try to use the following suggestion

```suggestion
        return StringUtils.isAnyEmpty(kol1.isEmpty(), kol2.isEmpty(), aktivnost.isEmpty(), labosi.isEmpty(), nazivKolegija.isEmpty(), nazivModela.isEmpty());
```